code/game/atoms.dm

```diff
@@ -1127,3 +1146,18 @@
 		var/atom/atom_orbiter = o
 		output += atom_orbiter.get_all_orbiters(processed, source = FALSE)
 	return output
+
+///Sets the custom materials for an item.
+/atom/proc/set_custom_materials(var/list/materials, multiplier = 1)
+	if(custom_materials) //Only runs if custom materials existed at first. Should usually be the case but check anyways
```

Comment:
And this comment says that the list should usually exist?

Reply:
> And this comment says that the list should usually exist?

It does exist in atoms.dm